content/en/docs/reference/setup-tools/kubeadm/kubeadm-upgrade.md

```diff
@@ -14,22 +14,25 @@ cluster if necessary.
 {{% /capture %}}
 
 {{% capture body %}}
+
 ## kubeadm upgrade guidance
 
 Every upgrade process might be a bit different, so we've documented each minor upgrade process individually.
 For more version-specific upgrade guidance, see the following resources:
 
- * [1.10 to 1.11 upgrades](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-11/)
- * [1.11 to 1.12 upgrades](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-12/)
  * [1.12 to 1.13 upgrades](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-13/)
- 
+ * [1.13 to 1.14 upgrades](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14/)
+ * [1.14 to 1.15 upgrades](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-15/)
+
 _For older versions, please refer to older documentation sets on the Kubernetes website._
 
 In Kubernetes v1.11.0 and later, you can use `kubeadm upgrade diff` to see the changes that would be
 applied to static pod manifests.
 
 To use kube-dns with upgrades in Kubernetes v1.13.0 and later please follow [this guide](docs/reference/setup-tools/kubeadm/kubeadm-init-phase/#cmd-phase-addon).
 
+## The kubeadm upgrade workflow
```

Comment:
Probably we should document that starting from 1.15 kubeadm upgrade apply and kubeadm upgrade node takes care of upgrading all the certificates, including certificates stored I. The kubeconfig files (link to the other doc)
The use can opt out from this behaviour with a new flag